src/app/pages/login/login-page/login-page.component.html

```diff
@@ -1 +1,53 @@
-<p>login-page works!</p>
+<div class="login-container">
+    <div class="left-side">
+        <h1>Connect with the most incredible professionals!</h1>
+    </div>
+    <div class="right-side">
+        <form
+            [formGroup]="loginForm"
+            (ngSubmit)="onSubmit()"
+            class="signup-form"
+        >
+            <h1 class="form-title">Login</h1>
+            <div class="username">
+                <label for="username">Email</label>
+                <input
+                    type="text"
+                    id="username"
+                    formControlName="username"
+                    required
+                />
+                <div
+                    class="username-errors"
+                    *ngIf="
+                        (loginForm.controls['username'].invalid && loginForm.controls.username.touched) ||
```

Comment:
Kindly use a getter to acquire the 'username' field in the class to make the code more readable. 
eg. get userName(){
return  this.loginForm.get('username')
}
then you chain the remaining properties you need to use on it.